doc/tutorials/using_windows_as_user_vm.rst

```diff
@@ -266,7 +266,7 @@ Explanation for acrn-dm Popular Command Lines
   This is GVT-d to passthrough the VGA controller to Windows.
   You may need to change 0/2/0 to match the bdf of the VGA controller on your platform.
 
-* ``-s 8,virtio-net,tap0``:
+* ``-s 8,virtio-net,tap=0``:
```

Comment:
```suggestion
* ``-s 8,virtio-net,tap=tap0``:
```

Reply:
As previous.
Could we use tap=LaaG to show the virtio-net format?

Reply:
change to tap=tap0